src/drivers/fluid_dsound.c

```diff
@@ -137,6 +155,7 @@ fluid_dsound_enum_callback2(LPGUID guid, LPCTSTR description, LPCTSTR module, LP
             return FALSE;
```

Comment:
`name` will be leaked when returning false here.

Reply:
fixed